### ISSUE TITLE
Auto-move container tab to panel doesn't work if "Reopen in container" context menu is used

### ISSUE THREAD

**Issue by jathek** [mbnuqw/sidebery#871]
## Expected behavior
Tab is moved to panel when reopened in a container

## Actual behavior
Tab doesn't move

## Debug info

https://user-images.githubusercontent.com/692914/204623063-fd58ffa2-5e74-4f25-b032-9bb49a7a2e2a.mp4

<details><summary><b>Addon data</b></summary>

```json
{
  "addonVersion": "5.0.0b29",
  "firefoxVersion": "108.0",
  "settings": {
    "nativeScrollbars": false,
    "nativeScrollbarsThin": true,
    "nativeScrollbarsLeft": false,
    "selWinScreenshots": false,
    "updateSidebarTitle": true,
    "markWindow": false,
    "markWindowPreface": "[Sidebery] ",
    "ctxMenuNative": false,
    "ctxMenuRenderInact": true,
    "ctxMenuRenderIcons": true,
    "ctxMenuIgnoreContainers": "",
    "navBarLayout": "horizontal",
    "navBarInline": true,
    "navBarSide": "left",
    "hideAddBtn": false,
    "hideSettingsBtn": false,
    "navBtnCount": true,
    "hideEmptyPanels": false,
    "navActTabsPanelLeftClickAction": "none",
    "navActBookmarksPanelLeftClickAction": "none",
    "navTabsPanelMidClickAction": "none",
    "navBookmarksPanelMidClickAction": "none",
    "navSwitchPanelsWheel": true,
    "groupLayout": "grid",
    "skipEmptyPanels": false,
    "dndTabAct": true,
    "dndTabActDelay": 750,
    "dndTabActMod": "none",
    "dndExp": "pointer",
    "dndExpDelay": 750,
    "dndExpMod": "none",
    "dndOutside": "win",
    "searchBarMode": "dynamic",
    "warnOnMultiTabClose": "collapsed",
    "activateLastTabOnPanelSwitching": true,
    "showTabRmBtn": true,
    "hideInact": false,
    "activateAfterClosing": "next",
    "activateAfterClosingGlobal": false,
    "activateAfterClosingNoFolded": true,
    "activateAfterClosingNoDiscarded": true,
    "askNewBookmarkPlace": true,
    "tabsRmUndoNote": true,
    "nativeHighlight": false,
    "tabsUnreadMark": false,
    "tabsUpdateMark": "all",
    "tabsReloadLimit": 5,
    "tabsReloadLimitNotif": true,
    "showNewTabBtns": true,
    "newTabBarPosition": "after_tabs",
    "tabsPanelSwitchActMove": false,
    "tabsUrlInTooltip": "full",
    "openSubPanelOnMouseHover": false,
    "colorizeTabs": false,
    "colorizeTabsSrc": "domain",
    "moveNewTabPin": "start",
    "moveNewTabParent": "last_child",
    "moveNewTabParentActPanel": false,
    "moveNewTab": "end",
    "moveNewTabActivePin": "start",
    "pinnedTabsPosition": "panel",
    "pinnedTabsList": false,
    "pinnedAutoGroup": false,
    "tabsTree": true,
    "groupOnOpen": true,
    "tabsTreeLimit": "none",
    "hideFoldedTabs": false,
    "autoFoldTabs": false,
    "autoFoldTabsExcept": "none",
    "autoExpandTabs": false,
    "rmChildTabs": "none",
    "tabsChildCount": true,
    "tabsLvlDots": true,
    "discardFolded": false,
    "discardFoldedDelay": 0,
    "discardFoldedDelayUnit": "sec",
    "tabsTreeBookmarks": true,
    "treeRmOutdent": "branch",
    "colorizeTabsBranches": false,
    "colorizeTabsBranchesSrc": "url",
    "warnOnMultiBookmarkDelete": "collapsed",
    "autoCloseBookmarks": false,
    "autoRemoveOther": false,
    "highlightOpenBookmarks": false,
    "activateOpenBookmarkTab": false,
    "showBookmarkLen": false,
    "bookmarksRmUndoNote": true,
    "loadBookmarksOnDemand": true,
    "pinOpenedBookmarksFolder": true,
    "loadHistoryOnDemand": true,
    "fontSize": "m",
    "animations": true,
    "animationSpeed": "norm",
    "theme": "proton",
    "density": "default",
    "colorScheme": "ff",
    "sidebarCSS": false,
    "groupCSS": false,
    "snapNotify": true,
    "snapExcludePrivate": false,
    "snapInterval": 0,
    "snapIntervalUnit": "min",
    "snapLimit": 0,
    "snapLimitUnit": "snap",
    "hScrollAction": "none",
    "navSwitchPanelsDelay": 128,
    "scrollThroughTabs": "none",
    "scrollThroughVisibleTabs": false,
    "scrollThroughTabsSkipDiscarded": false,
    "scrollThroughTabsExceptOverflow": true,
    "scrollThroughTabsCyclic": false,
    "scrollThroughTabsScrollArea": 0,
    "autoMenuMultiSel": true,
    "multipleMiddleClose": false,
    "longClickDelay": 500,
    "wheelThreshold": false,
    "wheelThresholdX": 10,
    "wheelThresholdY": 60,
    "tabDoubleClick": "none",
    "tabsSecondClickActPrev": true,
    "shiftSelAct": true,
    "activateOnMouseUp": false,
    "tabLongLeftClick": "none",
    "tabLongRightClick": "none",
    "tabCloseMiddleClick": "close",
    "tabsPanelLeftClickAction": "none",
    "tabsPanelDoubleClickAction": "tab",
    "tabsPanelRightClickAction": "menu",
    "tabsPanelMiddleClickAction": "tab",
    "newTabMiddleClickAction": "new_child",
    "bookmarksLeftClickAction": "open_in_act",
    "bookmarksLeftClickActivate": false,
    "bookmarksLeftClickPos": "default",
    "bookmarksMidClickAction": "open_in_new",
    "bookmarksMidClickActivate": false,
    "bookmarksMidClickPos": "default",
    "syncName": "",
    "syncSaveSettings": false,
    "syncSaveCtxMenu": false,
    "syncSaveStyles": false,
    "syncSaveKeybindings": false
  },
  "permissions": {
    "allUrls": false,
    "tabHide": false,
    "clipboardWrite": false,
    "webRequest": false,
    "webRequestBlocking": false
  },
  "storage": {
    "size": "6.93 kb",
    "props": {
      "containers": "1.25 kb",
      "favDomains": "64 b",
      "favHashes": "11 b",
      "favicons": "738 b",
      "prevTabsDataCache": "84 b",
      "profileID": "14 b",
      "settings": "3.63 kb",
      "sidebar": "785 b",
      "tabsDataCache": "254 b",
      "ver": "10 b"
    }
  },
  "sidebar": {
    "panels": {
      "tabs-panel": {
        "type": 2,
        "id": "tabs-panel",
        "name": "len: 4",
        "color": "toolbar",
        "iconSVG": "icon_tabs",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveTabCtx": "none",
        "moveTabCtxNoChild": true,
        "urlRulesActive": false,
        "urlRules": "",
        "bookmarksFolderId": -1,
        "newTabBtns": []
      },
      "Z-ql9srewnUe": {
        "type": 2,
        "id": "Z-ql9srewnUe",
        "name": "len: 4",
        "color": "orange",
        "iconSVG": "icon_flask",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveTabCtx": "firefox-container-1",
        "moveTabCtxNoChild": false,
        "urlRulesActive": false,
        "urlRules": "",
        "bookmarksFolderId": -1,
        "newTabBtns": []
      }
    },
    "nav": [
      "tabs-panel",
      "Z-ql9srewnUe",
      "add_tp",
      "sp-0",
      "settings"
    ]
  },
  "containers": [
    {
      "id": "firefox-container-1",
      "cookieStoreId": "firefox-container-1",
      "name": "8",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-2",
      "cookieStoreId": "firefox-container-2",
      "name": "4",
      "icon": "...",
      "color": "orange",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-3",
      "cookieStoreId": "firefox-container-3",
      "name": "7",
      "icon": "...",
      "color": "green",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-4",
      "cookieStoreId": "firefox-container-4",
      "name": "8",
      "icon": "...",
      "color": "pink",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    }
  ],
  "windows": [
    {
      "state": "maximized",
      "incognito": false,
      "tabsCount": 2
    }
  ],
  "bookmarks": "TypeError: can't access property \"getTree\", browser.bookmarks is undefined"
}
```
</details>

<details><summary><b>Logs</b></summary><code>
no error messages in console
</code></details>

**Comment by jathek** [comment]
I realize this behavior might be something some people prefer, so I would be fine with an option to toggle it.

**Comment by Redundakitties** [comment]
a workaround for this issue that I've found is using the extension [containerize ](https://addons.mozilla.org/en-US/firefox/addon/containerise/)
you have to play with it a bit but it seems to be properly move tabs between panels

**Comment by mbnuqw** [comment]
Fixed in v5.0.0b30.
